docs/project/tokenomics.mdx

```diff
@@ -1,80 +1,135 @@
+---
+id: tokenomics
+title: Tokenomics
+sidebar_label: Tokenomics
+slug: /tokenomics
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+ **WXM** is a an ERC20 token to power the weather data economy.
+
+:::note
+WeatherXM is based on an EVM compatible chain - we are currently developing on Polygon Testnet but this will be finalized near the token launch.
+:::
+
+### New approach in crowdsourcing weather data.
+WeatherXM creates an entirely new weather data economy that rewards the people who deploy and maintain weather stations.
+[WXM](https://mumbai.polygonscan.com/token/0xecbbbf304fdf91cddda6c6643ed1300a552b691a) is currently implemented in [Polygon Mumbai Testnet](https://mumbai.polygonscan.com/token/0xecbbbf304fdf91cddda6c6643ed1300a552b691a) as an ERC20 token rewarding the closed beta user group.
+
+
+
+Like most networks, the value of our Network is greater than the sum of its parts. Station owners are rewarded according to their contribution to the network and not based on the use of the specific data set they provided. 
+
+To implement this logic, we introduce a smart contract that distributes a fixed budget of tokens (the $WXM token) to station owners and other stakeholders daily. Any customer who wants to use the Network’s data or services will have to acquire and burn $WXM in return.
+
+The following sections describe how the smart contract issues new tokens, and how it distributes them to the Network contributors.
+
+## WXM Token Distribution
+
+**Total supply of the WXM token is 100M (100,000,000) tokens.**
+
+<div class="control">
+
+  <img src={useBaseUrl("img/project/tokenomics/wxm-distribution.png")} />
+ 
+</div>
+
+
+
+| Distribution               | Percentage  | Amount         |
+| ---------------------------| ----------- | -------------- |
+| **Station Rewards**      | 52%         | 52,000,000 WXM |
+| **Company & investors**    | 30%         | 30,000,000 WXM |
+| **Ecosystem Development**  | 10%         | 10,000,000 WXM |
+| **Liquidity Support**      | 5%          | 5,000,000 WXM  |
+|**"Airdrop" to beta participants**| 3%    | 3.000.000 WXM  |
+
+#### 55% - Station Rewards:
+tokens will be used for station owners rewards. 3M of this amount will be used to reward beta users.
+
+#### 30% - Company and Investors:
+will be used to reward the company and the investors for their contributions and work.
+
+#### 10% - Ecosystem Development:
+tokens will be reserved for ecosystem development and the formation of the DAO.
+
+#### 5% - Liquidity Bootstrap:
+will be used to bootstrap and support $WXM’s liquidity.
+
+#### 3% Airdrop" to beta participants
+3 million tokens will be used to reward stations that contributed data to the Network during the beta phase. This is not a marketing activity intended to create buzz and awareness, but a reward mechanism for the stations that participated and formed the Network before a reward mechanism was in place.

```

Comment:
Please after the `:` start the next sentence in the new line with an upper-case letter. Also the `:` is missing on the `3% Airdrop" to beta participants`